oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -112,6 +115,8 @@ public APIGatewayProxyResponseEvent handleRequest(
                                     ipAddress,
                                     AuditService.UNKNOWN,
                                     persistentSessionId);
+                            attachLogFieldToLogs(PERSISTENT_SESSION_ID, persistentSessionId);
```

Comment:
We need to add `.flatMap(InputSanitiser::sanitiseBase64)` to the method `getExistingOrCreateNewPersistentSessionId`. We did this in `PersistentIdHelper`.

Maybe there's some refactoring to do here.